src/plugins/data/public/utils/table_inspector_view/components/download_options.tsx

```diff
@@ -73,14 +71,14 @@ class DataDownloadOptions extends Component<DataDownloadOptionsProps, DataDownlo
         defaultMessage: 'unsaved',
       });
     }
-    exportAsCsv({
-      filename: `${filename}.csv`,
-      columns: this.props.columns,
-      rows: this.props.rows,
-      csvSeparator: this.props.csvSeparator,
-      quoteValues: this.props.quoteValues,
-      ...customParams,
-    });
+    // exportAsCsv({
```

Comment:
What's the deal here? Should this be put back or deleted?

Reply:
the PR is still work in progress